routing/routing_benchmarks/bicycle_routing_tests.cpp

```diff
@@ -44,4 +44,11 @@ UNIT_CLASS_TEST(BicycleTest, Smoke)
   m2::PointD const final = MercatorBounds::FromLatLon(55.79369, 37.56054);
   TestRouters(start, final);
 }
+
+UNIT_CLASS_TEST(BicycleTest, MailRuOfficeToUlSimanova5K2)
```

Comment:
Please, rename to Test1, or Moscow_Small1.
The reason is that MailRuOfficeToUlSimanova5K2 is a BAD address specification. Lat-lons is the test body a GOOD, but the name of the test is useless.